libraries/ui/src/main/java/androidx/media3/ui/PlayerView.java

```diff
@@ -1138,6 +1138,16 @@ public void setFullscreenButtonClickListener(@Nullable FullscreenButtonClickList
     controller.setOnFullScreenModeChangedListener(componentListener);
   }
 
+  /**
+   * Sets whether the fullscreen button is fullScreenButton or minimalFullScreenButton.
```

Comment:
Please see the difference between `fullScreenButton` and `minimalFullScreenButton`. They are not the same button. Both buttons act as fullscreen toggles and hence can have exit/enter fullscreen icons

<img width="613" alt="Screenshot 2024-09-04 at 00 01 56" src="https://github.com/user-attachments/assets/fabbe487-e682-43a6-974d-d158ba65f163">

Reply:
I rephrased in 2f1b08fa the javadoc:
Sets whether the player is currently in fullscreen, this will change the displayed icon.

If isFullscreen is true, ``exo_styled_controls_fullscreen_exit`` will be displayed or else ``exo_styled_controls_fullscreen_enter``.